src/cli.py

```diff
@@ -196,6 +199,13 @@ class Algorithm(StrEnum):
 Algorithms: PYRO, TANE
 Default: PYRO
 '''
+SFD_HELP = '''Discover soft functional dependencies. Soft functional dependencies are
```

Comment:
The SFD task can currently discover (1) correlations and (2) SFDs. Do you think we can split it into two different tasks? At least, I suggest to mention correlations in this help string.